client/ayon_core/hosts/max/plugins/publish/validate_mesh_has_uv.py

```diff
@@ -0,0 +1,44 @@
+
+import pyblish.api
+from ayon_core.hosts.max.api.action import SelectInvalidAction
+from ayon_core.pipeline.publish import (
+    ValidateMeshOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from pymxs import runtime as rt
+
+
+class ValidateMeshHasUVs(pyblish.api.InstancePlugin,
+                         OptionalPyblishPluginMixin):
+
+    """Validate the current mesh has UVs.
+
+    It validates whether the current mesh has texture vertex(s).
+    If the mesh does not have texture vertex(s), it does not
+    have UVs in Max.
+    """
+
+    order = ValidateMeshOrder
+    hosts = ['max']
+    families = ['model']
+    label = 'Validate Mesh Has UVs'
+    actions = [SelectInvalidAction]
+    optional = True
+
+
+    @classmethod
+    def get_invalid(cls, instance):
+        invalid = [member for member in instance.data["members"]
+                   if not member.mesh.numTVerts > 0]
+        return invalid
+
+    def process(self, instance):
+        invalid = self.get_invalid(instance)
+        if invalid:
+            raise PublishValidationError(
+                title="Mesh has missing UVs",
+                message="Model meshes are required to have UVs.<br><br>"
+                        "Meshes detected with invalid or missing UVs:<br>"
+                        "{0}".format([err.name for err in invalid])
+            )
```

Comment:
Is there a reason you're using html over regular line breaks?

I think the `<br>` might display ok in the UI but in logs will not be resolved to a new line and thus look weird there.
```suggestion
                message="Model meshes are required to have UVs.\n\n"
                        "Meshes detected with invalid or missing UVs:\n"
                        "{0}".format([err.name for err in invalid])
            )
```

Also, with this being an artist facing message we might want to turn it into bullet points OR at least `", ".join(names)` so they aren't printed as a Python list.